lib/gradebook.rb

```diff
@@ -0,0 +1,38 @@
+require 'pry'
+class Gradebook
+  attr_reader :name, :courses
+
+  def initialize(name)
+    @name = name
+    @courses = []
+
+  end
+
+  def add_course(course)
+    @courses << course
+  end
+
+  def list_all_students
+    students_per_course = {}
```

Comment:
What enumerable do you think you can use here so that you don't have to set an outside local variable in order to collect everything?